results/intfloat__multilingual-e5-small/ContractQALegalBenchPC.json

```diff
@@ -0,0 +1,49 @@
+{
+  "dataset_revision": "12ca3b695563788fead87a982ad1a068284413f4",
+  "mteb_dataset_name": "ContractQALegalBenchPC",
+  "mteb_version": "1.7.7",
+  "test": {
```

Comment:
Results are surprisingly high here, does this split contain negative samples along with positive ones?

Reply:
There are 41 negative samples and 39 positive samples in this dataset, so it's fairly balanced. I think due to the small size and relatively straightforward text samples, we are getting such high scores. You make a fair point that this task may be too trivial. I will remove this dataset, unless you have any other thoughts?

Reply:
Does positive and negative mean the same thing for all PC subsets? Maybe we can combine them to have 1 dataset with multiple samples and have 1 PC task. It'd possible only if the classes have the same meaning

Reply:
That's a good idea! For the added PC subsets (`ContractQA`, `CitationPrediction` and `ConsumerContractsQA`), the classes have the same meaning. We can combine them into one PC task.

Reply:
I am not sure how to combine the dataset, do I need to manually do it and push a new dataset to HF Hub and link to that instead of the original one or can we handle it in the class definition directly? How would I go about doing that?